content/3.Joining Corda Network/4.Onboarding Workflow.md

```diff
@@ -1,29 +1,35 @@
+If you are interested to join Corda Network, please send an email to info@corda.network with answers to these questions:
 
 1. Are you building / running a Corda application, or looking to use a Corda application?
 2. What is the use case and name (if already available) of your Corda application?
 3. Are you using Corda Open Source or Enterprise?
 4. Have you been in touch with any other R3 personnel?
 
-A member of our team will schedule a call with you to go through the onboarding process. We can also advise you on the network choice and contracting model. You can find more information about which Terms of Use agreement is most suitable for you on the Contracting Model session. 
+A member of our team will schedule a call with you to go through the onboarding process. We will also advise you on the network choice and contracting model.
 
 ## Step 2: Pre-approval of CSR information
 
-These steps ensures when the CSR is submitted (in Step 6), all values are populated correctly.
+Once you have signed the Corda Network Terms of Use, you are eligible to join the Corda Network! First you need to register your **identity** on the network (just like opening a bank account). As Corda is a permissioned blockchain network, your request to join the network will be reviewed and approved before your identity is registered on Corda Network.
 
-i. The node operator fills out fields in bold in the table below, for every node to be onboarded (you may submit as many nodes as you like). Submit your proposed field entries for review by raising a [Task](https://r3-cev.atlassian.net/servicedesk/customer/portal/2/group/2/create/79) ticket on the Support Portal.
+Please note that this step is a *MANDATORY STEP*, please ensure that you do this before sending any registration requests to the Corda Network Identity Manager. This step ensures when you submit your Certificate Signing Reqeuest ("CSR") in Step 6, your identity name is pre-approved already and the lead time for your identity to appear on the network map will be shortened.
 
-"O" - Organisation should usually be the same Legal Entity as what you have filled out in your Terms of Use (Step 1), which is also the owner of the assets being used on Corda Network. For guidance on which legal entity name to pick for your node, see the [Allowable Identity Names](https://corda.network/corda-network-rulebook/allowable-identity-names-on-corda-network) session.
+i. You will raise a [Task](https://r3-cev.atlassian.net/servicedesk/customer/portal/2/group/2/create/79) ticket on the Corda Network Support Portal. *Note: Your access to the Corda Network Support Portal will be granted after you signed the Corda Network Terms of Use, and your email address needs to be filled out on the Contact Details table on the Terms of Use. If you have any questions or needs access, please contact your account manager.*
```

Comment:
"or needs access" doesn't make sense

Reply:
What about changing it to "want to request access"?

Reply:
i would just keep as 'if you have any questions'